ci-scripts/copy_to_archive

```diff
@@ -33,42 +37,55 @@ fi
 
 # Copy the logs
 if [ -d ${LOG_DIR}/isafw-logs ]; then
-	echo "Archiving logs to $ARCHIVE_DIR"
-	mv ${LOG_DIR}/isafw-report_*/* $ARCHIVE_DIR
+    echo "Archiving logs to $ARCHIVE_DIR"
+    mv ${LOG_DIR}/isafw-report_*/* $ARCHIVE_DIR
+fi
+
+if [ -d ${LOG_DIR}/cooker ]; then
+    for f in ${LOG_DIR}/cooker/**/console-latest.log; do
+      echo "Archiving console-latest.log file to $ARCHIVE_DIR"
+      mv "$f" $ARCHIVE_DIR
+    done
+fi
+
+# Find the older directory
```

Comment:
What does `older` mean in this context?

Reply:
Directory which has been created first (read: dir created during build step)

Reply:
Consider rename `OLDER_DIR` to something like: `LICENSE_BUILD_DIR` or `LICENSE_IMAGE_DIR` or `LICENSE_IMAGE_BUILD_DIR`